guides/ReleaseNotes.md

```diff
@@ -1,5 +1,20 @@
 # Release Notes
 
+
+## New Release: [Wix Blog](wix-blog/blog)
+
+The [Blog](wix-blog/blog) API introduces new write functionality:
++ The new [Draft Posts](wix-blog/blog/draft-posts) API allows the ability to create, delete, and manage blog draft posts.
++ The new [Create Category](wix-blog/blog/categories/create-category) allows for creating a blog category.
```

Comment:
I would combine the category ones and then combine the tags one.  We don't need a bulleted list for each individual endpoint.  And we don't have to say "new" each time, it's implied. 

Is there a Categories API or are they just individual flat endpoints? 

And then something similar for tags. 

So something like "You can now manage blog categories using the Categories APIs, such as Update Category, Delete Category, and Update Category.